js/adapt-homeButton.js

```diff
@@ -33,41 +33,42 @@ define([
         },
 
         _disabled: function() {
-            this._$html.removeClass("hide-home-button");
+            this._$html.removeClass("hide-nav-home-btn");
             if (this._dataEvent) {
-                $(".navigation-home-button").attr("data-event", this._dataEvent);
+                $(".js-nav-home-btn").attr("data-event", this._dataEvent);
                 this._dataEvent = null;
             }
         },
 
         _enabled: function() {
-            this._$html.toggleClass("hide-home-button", !!this._config._hideHomeButton);
+            this._$html.toggleClass("hide-nav-home-btn", !!this._config._hideHomeButton);
             // extend functionality to toggle back button display
-            this._$html.toggleClass("hide-back-button", !!this._config._hideBackButton);
-            if (!$(".navigation-home-button")[0]) {
+            this._$html.toggleClass("hide-nav-back-btn", !!this._config._hideBackButton);
+            if (!$(".js-nav-home-btn")[0]) {
                 // if home button doesn't exist create home button
                 this._createHomeButton();
             }
             if (this._config._redirectToId) {
-                this._dataEvent = $(".navigation-home-button").attr("data-event");
-                $(".navigation-home-button").attr("data-event", "redirectedHomeButton");
+                this._dataEvent = $(".js-nav-home-btn").attr("data-event");
+                $(".js-nav-home-btn").attr("data-event", "redirectedHomeButton");
             }
         },
 
         _createHomeButton: function() {
             var config = Adapt.course.get("_homeButton");
             var altText = (config && config.alt);
             var $backButton = $('button[data-event="backButton"]');
+            var $icon = $('<div>', {'class': 'icon'});
             var $homeButton = $('<button>', {
                 attr: {
                     'data-event': 'homeButton'
                 },
-                'class': 'base navigation-home-button icon icon-home',
-                'aria-label': altText
-            });
+                'class': 'btn-icon nav__btn nav__home-btn js-nav-home-btn',
+                'aria-label': altText,
+                'role' : 'link'
```

Comment:
Don't need quotes around `role`:
```suggestion
                role: 'link'